arch/arm/boot/dts/qcom/msm8939.dtsi

```diff
@@ -27,7 +27,7 @@
 	qcom,msm-id = <239 0>, <241 0>, <263 0>;
 
 	chosen {
-		bootargs = "boot_cpus=0,4,5,6,7 sched_enable_hmp=1";
+		bootargs = "boot_cpus=0,1,2,3,4,5,6,7 sched_enable_hmp=1";
```

Comment:
wait...
what is the real function of sched_enable_hmp??

removing this command can help boosting boot time